### PR TITLE
Changes to API and API client

### PR DESCRIPTION
**Pull request by kiddinn**
This is a mixed bag PR that includes few changes:

1. API Client: analyzer object is now easier to parse for tools using the API to run analyzer(s)
2. API Client: Search now has support for defining the indices
3. API Client: The timeline object now has an additional method for running several analyzers `run_analyzers`
4. Additional error handling in the API
5. Change to the analyzer interface, to change the flush interval for events
6. Few more error handling in various places
7. Minor fix in the importer client and a version bump to come with it

### CODE REVIEW

File: timesketch/lib/analyzers/interface.py [hunk 1]
```diff
@@ -113,7 +127,7 @@ class Event(object):
         index_name: The name of the Elasticsearch index.
         source: Source document from Elasticsearch.
     """
-    def __init__(self, event, datastore, sketch=None):
+    def __init__(self, event, datastore, sketch=None, analyzer=None):
```
Comment (berggren) [inline timesketch/lib/analyzers/interface.py:130]: Add analyzer to Args docstring

**Review by berggren — APPROVED** [review]
Just a tiny small missing docstring arg...... otherwise LGTM

File: timesketch/lib/analyzers/interface.py (hunk 1 above)
Comment (kiddinn) [reply timesketch/lib/analyzers/interface.py:130]: done

**Review by kiddinn — COMMENTED** [review]
done with round

**Pull request merged**